CRM/Core/BAO/CustomField.php

```diff
@@ -1158,7 +1158,7 @@ public static function displayValue($value, $field, $entityId = NULL) {
       $field = self::getFieldObject($fieldId);
     }
 
-    $fieldInfo = array('options' => $field->getOptions()) + (array) $field;
+    $fieldInfo = array('options' => $field->getOptions($context = $context)) + (array) $field;
```

Comment:
I think you should pass "get" here instead of `$context = $context` (which is redundant anyway)

Reply:
Won't this display the disabled fields on multiple places?
Or is that the desired behaviour?